README.md

```diff
@@ -61,14 +37,27 @@ terraform {
   required_providers { iterative = { source = "iterative/iterative" } }
 }
 provider "iterative" {}
-# ... other resource blocks ...
+resource "iterative_task" "example" {
+  cloud   = "aws" # or any of: gcp, az, k8s
+  machine = "m"   # medium, or any of: l, xl, m+k80, xl+v100, ...
+
+  storage {
+    workdir = "."
+    output  = "results"
+  }
+  script = <<-END
+    #!/bin/bash
+    mkdir results
+    echo "Hello World!" > results/greeting.txt
+  END
+}
 ```
 
-**Note:** to use your local build, specify `source = "github.com/iterative/iterative"` (`source = "iterative/iterative"` will download the latest stable release instead).
+See the [Reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task) for the full list of options for `main.tf`.
```

Comment:
Should the link point directly to References? https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#argument-reference